app/_hub/kong-inc/openid-connect/index.md

```diff
@@ -1,116 +1,121 @@
-  service protected with the OpenID Connect plugin (or you can do both).
-
-  Some capabilities of the plugin are listed below:
-
-  - [WebFinger][webfinger] and [OpenID Connect Discovery][discovery]
-  - [ID Token][idtoken] verification
-  - [UserInfo][userinfo] endpoint data injecting
-  - [RP-Initiated Logout][rplogout]
-  - [OAuth 2.0 Token Revocation][revocation] during the logout (optionally)
-  - [OAuth 2.0 Token Introspection][introspection] support
-  - [OAuth 2.0 Proof Key for Code Exchange][pkce] (PKCE) support
-  - Standard and configurable claims verification
-  - Caching (optional) of token, introspection and user info endpoint request
-
-  The plugin has been tested with several OpenID Connect capable providers, such as:
-
-  - [Auth0][auth0]
-  - [Amazon AWS Cognito][cognito]
+  identity provider (IdP) in a standardized way. This plugin can be used to implement
+  Kong as a (proxying) [OAuth 2.0][oauth2] resource server (RS) and/or as an OpenID
+  Connect relying party (RP) between the client, and the upstream service.
+
+  The plugin supports several types of credentials and grants:
+
+  - Signed [JWT][jwt] access tokens ([JWS][jws])
+  - Opaque access tokens
+  - Refresh tokens
+  - Authorization code  
+  - Username and password
+  - Client credentials
+  - Session cookies
+
+  The plugin has been tested with several OpenID Connect providers:
+
+  - [Auth0][auth0] ([Kong Integration Guide](/enterprise/latest/plugins/oidc-auth0))
+  - [Amazon AWS Cognito][cognito] ([Kong Integration Guide](/enterprise/latest/plugins/oidc-cognito/))
   - [Connect2id][connect2id]
+  - [Curity][curity]
   - [Dex][dex]
   - [Gluu][gluu]
-  - [Google][google]
-  - [IdentityServer4][identityserver4]
+  - [Google][google] ([Kong Integration Guide](/enterprise/latest/plugins/oidc-google/))
+  - [IdentityServer][identityserver]
   - [Keycloak][keycloak]
-  - [Microsoft Azure Active Directory v1][azurev1]
-  - [Microsoft Azure Active Directory v2][azurev2]
-  - Microsoft Live Connect
-  - [Okta][okta]
+  - [Microsoft Azure Active Directory][azure] ([Kong Integration Guide](/enterprise/latest/plugins/oidc-azuread))
+  - [Microsoft Active Directory Federation Services][adfs]
+  - [Microsoft Live Connect][liveconnect]
+  - [Okta][okta] ([Kong Integration Guide](/enterprise/latest/plugins/oidc-okta))
   - [OneLogin][onelogin]
   - [OpenAM][openam]
   - [Paypal][paypal]
   - [PingFederate][pingfederate]
   - [Salesforce][salesforce]
+  - [WSO2][wso2]
   - [Yahoo!][yahoo]
 
-  As long as your provider supports OpenID Connect standards, the plugin should
-  work, even if it is not specifically tested against it. Let Kong know if you
-  want your provider to be tested and added to the list.
+  As long as your provider supports OpenID Connect, OAuth or JWT standards,
+  the plugin should work, even if it is not specifically tested against it.
+  Let Kong know if you want your provider to be tested and added to the list.
```

Comment:
@HCloward what do you think: should we remove that line or add instructions? In reality it can come as a pr to docs repo, or an issue to docs repo? Perhaps there are other ways too. Both online and offline.

Reply:
If they make a PR to the docs repo, who will test it? I do not think the docs team will be able to test these scenarios.

Reply:
@HCloward, I assumed the the third party had tested it. I will remove the line as otherwise I need to answer more questions.